hazenlib/__init__.py

```diff
@@ -298,14 +298,35 @@ def get_field_of_view(dcm: pydicom.Dataset):
 
     return fov
 
+  
+def is_dicom_file(filename):
```

Comment:
Can we have a test function that shows this works in an associate test file?

Reply:
@hshuaib90 added, let me know if it is ok!